src/test/java/uk/gov/pay/connector/gateway/worldpay/wallets/WorldpayWalletAuthorisationHandlerTest.java

```diff
@@ -101,7 +102,7 @@ public void shouldSendGooglePayRequestWhenGooglePayDetailsArePresent() throws Ex
     @Test
     public void shouldSendGooglePay3DSRequestWhenGooglePayDetailsWithoutIpAddressArePresent() throws Exception {
         try {
-            worldpayApplePayAuthorisationHandler.authorise(getGooglePay3DSAuthorisationRequestWithoutIpAddress());
+            worldpayApplePayAuthorisationHandler.authorise(getGooglePay3DSAuthorisationRequest(true, false));
```

Comment:
Can we use `3ds` instead of `3DS` for consistency?

```suggestion
            worldpayApplePayAuthorisationHandler.authorise(getGooglePay3dsAuthorisationRequest(true, false));
```

Reply:
Done.